akka-http/src/main/scala/akka/http/scaladsl/server/RejectionHandler.scala

```diff
@@ -165,7 +165,11 @@ object RejectionHandler {
       }
       .handle {
         case TooManyRangesRejection(_) ⇒
-          complete((RequestedRangeNotSatisfiable, "Request contains too many ranges."))
+          complete((RequestedRangeNotSatisfiable, "Request contains too many ranges"))
+      }
+      .handle {
+        case CircuitBreakerOpenRejection(_) ⇒
+          complete((EnhanceYourCalm, "Circuit Breaker is open; calls are failing fast"))
```

Comment:
Still haven't changed this as EnhanceYourCalm was the original proposal, but I see your point. We could reject with ISE, anyway custom handlers can be setup on top of it..
Any of the project members agree?